### DISCUSSION TITLE
How do I control the distance at which details are culled?

### DISCUSSION THREAD

**Discussion by HellbourneAL** [visualdesigncafe/nature-renderer#82] [Help]
Little more explanation, I am making an isometric topdown game, I have pebbles on the ground, this pebbls will shrink from nothing to full size on screen, how do I control when this is being done via distance?

**Comment by HellbourneAL** [comment]
![image](https://github.com/user-attachments/assets/b4481315-3776-4b05-b967-d93cdf18e679)
![image](https://github.com/user-attachments/assets/cb489217-772d-4612-b516-b1d73df17df5)

**Reply by HellbourneAL** [reply]
Notice the extra pebble in the second screenshot, that one just appears as the camera moves closer its very noticeable

**Comment by maxplooi** [comment]
This is done with the **Density in Distance** setting in either the [Object Settings](https://visualdesigncafe.com/nature-renderer/docs/2022/object-settings), [Camera Settings](https://visualdesigncafe.com/nature-renderer/docs/2022/camera-settings), or [Default Settings](https://visualdesigncafe.com/nature-renderer/docs/2022/default-settings). Setting the value to 1 will disable the fading completely.
